go/pkg/repositories/application_logs_repository.go

```diff
@@ -96,6 +99,49 @@ func (r *ElasticSearchApplicationLogsRepository) doesIndexExists(index string) b
 	return r.indices[index]
 }
 
+func (r *ElasticSearchApplicationLogsRepository) GetLogsByIds(
+	ctx context.Context,
+	clusterId string,
+	startDate time.Time,
+	endDate time.Time,
+	ids []string,
+) ([]*ApplicationLogsDocument, error) {
+
+	indices := r.getIndiciesWithClusterAndDateRange(clusterId, startDate, endDate)
+
+	logsByIdsQuery, err := elasticsearch.GetDocumentsByIds(ctx,
+		r.esClient,
+		indices,
+		ids)
+	if err != nil {
+		r.logger.Error("Failed to get logs by query", zap.Error(err), zap.Any("ids", ids), zap.Any("clusterId", clusterId))
+		return nil, err
+	}
+
+	var applicationLogs []*ApplicationLogsDocument
```

Comment:
This mapping is very similar for both nodes and apps, maybe its worth extracting to some utils?